plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -233,6 +235,11 @@ static obs_properties_t *ffmpeg_source_getproperties(void *data)
 static void dump_source_info(struct ffmpeg_source *s, const char *input,
 			     const char *input_format)
 {
+	if (!s->log_changes) {
+		UNUSED_PARAMETER(input);
+		UNUSED_PARAMETER(input_format);
+		return;
+	}
```

Comment:
```suggestion
	if (!s->log_changes)
		return;
```
`UNUSED_PARAMETER` is a macro to suppress a warning that the parameter is not used in the function. We don't need to have this in the `if` block.

Reply:
Resolved, thank you!